FireRender.Maya.Src/FireRenderProduction.cpp

```diff
@@ -1065,6 +1066,21 @@ void FireRenderProduction::RenderTiles()
 		data = it->second.get();
 	}
 
+	// run merge opacity
+	m_contextPtr->ProcessMergeOpactityFromRAM(data, info.totalWidth, info.totalHeight);
+
+	// apply render stamp
+	FireMaya::RenderStamp renderStamp;
+	MString stampStr;
+	m_aovs->ForEachActiveAOV([&](FireRenderAOV& aov)
```

Comment:
Why you iterate through all aovs using this ruotine ?
Please use  the following:
FireRenderAOV* getAOV(unsigned int id);

Reply:
done